### PR TITLE
feat: add conformance test

### PR DESCRIPTION
**Pull request by vishal-chdhry**

### CODE REVIEW

File: .github/workflows/tests.yaml [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  chainsaw:
+    strategy:
+      fail-fast: false
+      matrix:
+        k8s-version:
```
Comment (realshuting) [inline .github/workflows/tests.yaml:20]: Should we target k8s 1.26-1.29?

File: .github/workflows/tests.yaml [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+          - name: v1.25
+            version: v1.25.11
+          - name: v1.26
+            version: v1.26.6
+          - name: v1.27
+            version: v1.27.3
+          - name: v1.28
+            version: v1.28.0
+        tests:
+          # - ^autogen$
```
Comment (realshuting) [inline .github/workflows/tests.yaml:30]: Can we remove un-used ones?

File: config/debug.yaml
Comment (realshuting) [inline config/debug.yaml:1]: Do we need to check this in?

Comment (vishal-chdhry) [reply config/debug.yaml:1]: Currently, the debug manifest uses latest and the install manifest uses a release version (we don't have that yet).
I think I should just build images from scratch using makefile, I will update that

**Review by realshuting — DISMISSED** [review]

File: config/debug.yaml
Comment (vishal-chdhry) [reply config/debug.yaml:1]: @realshuting I have removed the debug manifest and have moved to building the current codebase in each PR

**Pull request merged**